docs/source/quick_start.rst

```diff
@@ -0,0 +1,181 @@
+Quick Start
+============
+
+This guide is intended to jump start new Kokkos users (and beginners, in particular).  Kokkos Core is the foundation for all other Kokkos Ecosystem projects, and is readily usable in your own for performance portability.
+
+
+Download Latest and Build 
+-----------------------------
+
+.. note::
+
+  Please become familiar with `Kokkos Requirements <https://kokkos.org/kokkos-core-wiki/requirements.html>`_, and verify that your machine has all necessary compilers, backend SDK and build system components.
+
+
+..
+ Nota bene:  the link for "Latest" should be stable from one release to the next, but check periodically to be sure 
+..
+
+:bdg-link-primary:`Latest Release <https://github.com/kokkos/kokkos/releases/latest>`
+
+.. code-block:: sh
+  
+  # Uncomment according to the type of file you've downloaded (zip or tar)
+  unzip kokkos-x.y.z.zip 
+  # tar -xzf kokkos-x.y.z.tar.gz
```

Comment:
```suggestion
  tar xzf kokkos-x.y.z.tar.gz
```

I think we can focus on one type of file, not having to add 2 other lines that make thing looks complicated.

Reply:
I put zip and tar.gz, because those are the choices you are presented on the latest release.  Please keep in mind that the "Quick Start" needs to be geared for the near-absolute beginner.